app/models/converters/boolean_converter.rb

```diff
@@ -5,6 +5,6 @@ class BooleanConverter < BaseConverter
 
   def self.convert(value)
     value = super(value)
-    value.blank? ? nil : TRUTHS.include?(value.downcase)
+    value.blank? ? nil : TRUTHS.include?(value.to_s)
```

Comment:
So I believe @typesend introduces virtus into this project with his PR, would love to maybe see this type of thing replaced with a custom coercion class. We can revisit this in another PR.